test/data/sandbox/configs/pageObjects/first_test.js

```diff
@@ -1,12 +1,12 @@
 Feature('PageObject');
 
-Scenario('@ClassPageObject', async (I, classpage) => {
+Scenario('@ClassPageObject', async ({ I, classpage }) => {
```

Comment:
if there is way to to input with or without curly brackets would be nice, users don't need to migrate their current scripts ^^

Reply:
We will provide a migration script for them.